loris/img_info.py

```diff
@@ -325,9 +331,15 @@ def get(self, ident):
 
                 lastmod = datetime.utcfromtimestamp(os.path.getmtime(info_fp))
                 info_and_lastmod = (info, lastmod)
-                logger.debug('Info for %s read from file system', ident)
                 # into mem:
                 self.__setitem__(ident, info, _to_fs=False)
+
+        #check that src image exists if it's in the info
```

Comment:
This comment wasn't very clear to me when I first read it. Maybe:

> If a source image is referred to in a cached info.json, check it exists on disk.

Would we expect to have an info.json that refers to a non-existent image? Maybe log a warning in that case?